catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -289,14 +295,31 @@ private void createThumbnail(InputStream input, Metacard metacard) {
 
     private String transformToXml(String xhtml) {
         LOGGER.debug("Transforming xhtml to xml.");
+
+        XMLReader xmlReader = null;
         try {
-            Writer xml = new StringWriter();
-            Transformer transformer = templates.newTransformer();
-            transformer.transform(new StreamSource(new StringReader(xhtml)), new StreamResult(xml));
-            return xml.toString();
-        } catch (TransformerException e) {
-            LOGGER.debug("Unable to transform metadata from XHTML to XML.", e);
-            return xhtml;
+            XMLReader xmlParser = XMLReaderFactory.createXMLReader();
+            xmlParser.setFeature("http://xml.org/sax/features/external-general-entities", false);
+            xmlParser.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+            xmlParser.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd",
+                    false);
+            xmlReader = new XMLFilterImpl(xmlParser);
+        } catch (SAXException e) {
+            LOGGER.debug(e.getMessage(), e);
+        }
+        if (xmlReader != null) {
+            try {
+                Writer xml = new StringWriter();
+                Transformer transformer = templates.newTransformer();
+                transformer.transform(
+                        new SAXSource(xmlReader, new InputSource(new StringReader(xhtml))),
+                        new StreamResult(xml));
+                return xml.toString();
+            } catch (TransformerException e) {
+                LOGGER.debug("Unable to transform metadata from XHTML to XML.", e);
+                return xhtml;
```

Comment:
You should be able to remove this return statement; it'll fall through to line 323.